ts/utils/supportAssistance.ts

```diff
@@ -10,19 +12,23 @@ export type ZendeskConfig = {
 export type JwtIdentity = ZendDesk.JwtIdentity;
 export type AnonymousIdentity = ZendDesk.AnonymousIdentity;
 
-export const zendeskDefaultJwtConfig: ZendeskConfig = {
+export const zendeskDefaultJwtConfig: ZendeskAppConfig = {
   key: "mp9agCp6LWusBxvHIGbeBmfI0wMeLIJM",
   appId: "7f23d5b0eadc5b4f2cc83df3898c6f607bad769fe053a186",
   clientId: "mobile_sdk_client_4bf774ed28b085195cc5",
   url: "https://appiotest.zendesk.com"
 };
-export const zendeskDefaultAnonymousConfig: ZendeskConfig = {
+export const zendeskDefaultAnonymousConfig: ZendeskAppConfig = {
   key: "mp9agCp6LWusBxvHIGbeBmfI0wMeLIJM",
   appId: "8547479b47fdacd0e8f74a4fb076a41014dee620d1d890b3",
   clientId: "mobile_sdk_client_518ee6a8160220698f97",
   url: "https://appiotest.zendesk.com"
 };
 
+// If is not possible to get the Zendesk remote config assume it as active.
+export const isZendeskActiveRemotely = (zRC: ZendeskConfig | undefined) =>
```

Comment:
In this case I strongly recommend the opposite:
if the app doesn't know about the remote FF is more safe to assume it is `false`

take in consideration this scenario: we have a critical problem with zendesk, then we put it off the remote FF. If a user uses the app in that moment and for some reasons it can't get the remote data (in the mobile scenario all requests could fail for multiple reasons - i.e put the device offline), then the user can see zendesk while we don't want 🚨 

what do you think?

Reply:
My idea was to set it to `true` given the limitations we will have on Instabug.
I imagined this scenario: the app fails to retrieve this data, there are actually no issues on Zendesk, all reports will go to Instabug.

Reply:
@Undermaken the logic has been changed by choosing to use a default "none" value which does not show the assistance button.